contracts/src/v0.8/functions/tests/1_0_0/testhelpers/FaultyFunctionsClientTestHelper.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import {ITermsOfServiceAllowList} from "../../../dev/1_0_0/accessControl/interfaces/ITermsOfServiceAllowList.sol";
+import {IFunctionsSubscriptions} from "../../../dev/1_0_0/interfaces/IFunctionsSubscriptions.sol";
+
+import {FunctionsRequest} from "../../../dev/1_0_0/libraries/FunctionsRequest.sol";
+import {FunctionsClient} from "../../../dev/1_0_0/FunctionsClient.sol";
+
+contract FaultyFunctionsClientTestHelper is FunctionsClient {
```

Comment:
What's faulty about it? Empty callback?

Reply:
Faulty might not the right word.

An empty callback means that the user's client contract doesn't do anything with the data it receives, which at the time I wrote this seemed to not be a valid behavior.

I suppose that maybe the user could be just monitoring for events off-chain?

Reply:
How about I call it empty instead?

Reply:
Yeah maybe FunctionsClientWithEmptyCallback?